src/Evolution/DgSubcell/Tags/Inactive.hpp

```diff
@@ -0,0 +1,43 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include "DataStructures/DataBox/PrefixHelpers.hpp"
+#include "DataStructures/DataBox/Tag.hpp"
+#include "DataStructures/VariablesTag.hpp"
+#include "Evolution/DgSubcell/ActiveGrid.hpp"
+
+/// \cond
+template <typename TagsList>
+class Variables;
+/// \endcond
+
+namespace evolution::dg::subcell::Tags {
+namespace detail {
+// Struct used to mark the inactive grid quantities for easy typelist
+// manipulations such as filtering out all inactive tags.
```

Comment:
is this somehow better than filtering on `tt::is_a<Tags::Inactive>` ?

Reply:
No idea... I'll remove it for now. I'm not sure I'll even need to filter on that anymore.